src/tests/multihost/alltests/test_misc.py

```diff
@@ -353,3 +353,48 @@ def test_0006_getent_group(self, multihost,
         assert "group-2@example1:*:20002:user-2@example1," \
                "user-4@example1,user-6@example1," \
                "user-8@example1" in cmd.stdout_text
+
+    @pytest.mark.tier1
```

Comment:
Change tier1 to tier1_2

Reply:
I've made the change and commited. Thanks.